src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -298,13 +303,22 @@ bool clang_c_convertert::get_struct_union_class(const clang::RecordDecl &rd)
   if(context.find_symbol(id) != nullptr)
     return false;
 
+  // TODO: Fix me when we have a test case using C++ unioin.
+  //       A C++ union can have member functions but not virtual functions.
+  //       Just use struct_typet for C++?
   struct_union_typet t;
   if(rd.isUnion())
     t = union_typet();
   else
     t = struct_typet();
   t.tag(name);
 
+  if(rd.isClass())
```

Comment:
I suppose so... just need to distinguish class from struct, making it more intuitive in the symbol table, otherwise it would mark everything `struct`:
![Screenshot 2023-01-25 at 12 51 00](https://user-images.githubusercontent.com/32592800/214568146-ba4906d5-eeaa-4523-b136-e015dcbfc45f.png)

Reply:
If it's just to print "class" instead of "struct", we don't really need it.

Reply:
done removed